.vscode/settings.json

```diff
@@ -54,5 +54,6 @@
     "files.eol": "\n",
     "typescript.unstable": {
         "organizeImportsIgnoreCase": true
-    }
+    },
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Why is this required?

Reply:
VS Code is still shipped with ts 5.4 and this will use the locale ts version.

Reply:
according vscode docs it should ship with latest ts, but maybe it will change to 5.5 in the next version..